Fix Fantem ZB006-X no response recognized for changing device settings

AlexKay88 opened the pull request:
Trying to fix recognizing responses for changing device settings.
Device settings e.g. ext_switch_type, load_detection_mode and control_mode can be changed successfully.
But responses from device are not recognized, because it sends them with unsupported command 0x05.

Already introduced support in https://github.com/Koenkk/zigbee-herdsman/pull/519 for command 0x05 by adding a new type commandActiveStatusReportAlt


Need further testing and dont want to break something, thus copied tuya_data_point_dump
into  tuya_data_point_dump_alt (cluster: 'manuSpecificTuya', type: ['commandActiveStatusReportAlt').

Koenkk commented on converters/fromZigbee.js:
```diff
@@ -4726,6 +4726,43 @@ const converters = {
+                dataStr += '\n';
+            }
+            const fs = require('fs');
+            fs.appendFile('data/tuya.dump.alt.txt', dataStr, (err) => {
```
Please remove this, can cause issues on systems which are not writeable

AlexKay88 replied:
Tuya dump write to file is for debugging only, thus not meant for production systems and debugging is usually not done on read-only systems.
Same file writes are in function `tuya_data_point_dump` right above in code. Also logs to file and is for debugging only.
Would be very helpful for further debugging if this could stay in code.
Pls. consider merging.

Koenkk replied:
But if I merge this it will be used in production, so therefore I cannot merge it in this state.

AlexKay88 replied:
Removed write to fs

Koenkk commented:
Note to self; merge https://github.com/Koenkk/zigbee-herdsman/pull/519 after this.

AlexKay88 commented on converters/fromZigbee.js:
```diff
@@ -4726,6 +4726,18 @@ const converters = {
             });
         },
     },
+    tuya_data_point_dump_alt: {
```
It was intended to enable `tuya_data_point_dump_alt` on my local installation for debugging, but of course I can enable it for everyone. Pls. find next commit below.

Koenkk replied:
Could we maybe already integrate `tuya_data_point_dump_alt` into  `tuya_data_point_dump`?

AlexKay88 replied:
Not now, because it needs debugging to be sure if decoding received data in `cluster.ts` `activeStatusReportAlt` is really decodeable with `uint16 `and `BuffaloZclDataType`
```
                    {name: 'seq', type: DataType.uint16},
                    {name: 'dpValues', type: BuffaloZclDataType.LIST_TUYA_DATAPOINT_VALUES},
```
My intention from the beginning was to add **minimal code changes** to
- support new 0x05 command in `cluster.ts` `activeStatusReportAlt`
- add `activeStatusReportAlt` to `fromZigbee ``tuya_data_point_dump_alt`
- simply to be able to proceed with debugging on my local system with `tuya_data_point_dump_alt`
- in order to prepare production-ready code changes with a future PR

I did not want to break anything existing, thats why I did not at once add
- `activeStatusReportAlt` to existing `fromZigbee` `tuya_data_point_dump`
- `activeStatusReportAlt` to existing `fromZigbee` `ZB006X_settings`
- `fz.tuya_data_point_dump_alt` to existing `devices/fantem.js` 
- and instead copied `tuya_data_point_dump` into `tuya_data_point_dump_alt`

My goal was just beeing able to debug, proof my local code is working fine and **afterwards**
- `activeStatusReportAlt` to existing `fromZigbee` `tuya_data_point_dump`
- `activeStatusReportAlt` to existing `fromZigbee` `ZB006X_settings`
- remove `fromZigbee ``tuya_data_point_dump_alt` again.

If you show me a way to add 0x05 command in `cluster.ts` `activeStatusReportAlt` by changes to my local filesystem, I will stop asking for PRs.

Koenkk replied:
I've merged https://github.com/Koenkk/zigbee-herdsman/pull/519 now, so you can keep the changes in this PR in your local external converter. Once production code is ready you can update this PR.

Changes will be available in the dev branch in a few hours from now. (https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html)

AlexKay88 commented:
New code for `cluster 0xef00 cmd 0x05 commandActiveStatusReportAlt` is running fine.
Details about merged code: https://github.com/Koenkk/zigbee-herdsman/pull/519 

Finalized changes for merging:
- removed temporary `fromZigbee/tuya_data_point_dump_alt`
- added `commandActiveStatusReportAlt` to `fromZigbee/tuya_data_point_dump`
- added `commandActiveStatusReportAlt` to `fromZigbee/ZB006X_settings`
- added `commandActiveStatusReport` to `fromZigbee/ZB006X_settings` (cmd 0x06 is used, too)
- fixed `toZigbee/ZB006X_settings` to use `cmd 0x04` instead of `0x00`
- exposed new functionality `power_supply_mode, ext_switch_status, load_type, load_dimmable`
- [Enabled reporting of physical state changes](https://github.com/Koenkk/zigbee2mqtt/issues/9057#issuecomment-1007742130)

AlexKay88 commented:
@Koenkk I would be glad if you could review and perhaps merge, if code is fine. THX!
